src/Components/Facility/HospitalList.tsx

```diff
@@ -105,21 +105,25 @@ export const HospitalList = () => {
   };
   const onSearchSuspects = async (searchValue: string) => {
     setIsLoading(true);
-    const res = await dispatchAction(getFacilities({ limit, offset, search_text: searchValue }));
+    const res = await dispatchAction(
+      getFacilities({ limit, offset, search_text: searchValue })
```

Comment:
Codacy found an issue: [Identifier 'search_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5830437)